osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -180,5 +184,29 @@ public void OnReleased(KeyBindingReleaseEvent<ManiaAction> e)
         public override bool ReceivePositionalInputAt(Vector2 screenSpacePos)
             // This probably shouldn't exist as is, but the columns in the stage are separated by a 1px border
             => DrawRectangle.Inflate(new Vector2(Stage.COLUMN_SPACING / 2, 0)).Contains(ToLocalSpace(screenSpacePos));
+
+        #region Touch Input
+
+        [Resolved(canBeNull: true)]
+        private ManiaInputManager maniaInputManager { get; set; }
```

Comment:
can be null?

Reply:
Can be null according to some mania editor tests that initialise `Column`s without surrounding input managers. I've enabled NRT to fix usages bypassing this.